Composer/packages/tools/language-servers/language-generation/package.json

```diff
@@ -8,7 +8,7 @@
     "build:demo": "cd demo && tsc --build tsconfig.json",
     "prepublishOnly": "npm run build",
     "clean": "rimraf lib demo/dist",
-    "start": "cd demo && set NODE_ENV=test&& ts-node ./src/server.ts",
+    "start": "cd demo && cross-env NODE_ENV=test ts-node ./src/server.ts",
```

Comment:
does `cross-env` need to be added as a dev dependency?

Reply:
cross-env has added in root package.json now